toctoc/slickMySql/src/test/scala/io.buildo.toctoc/slick/authentication/test/MySqlSlickLoginAuthenticationDomainFlowSpec.scala

```diff
@@ -49,66 +50,123 @@ class MySqlSlickLoginAuthenticationDomainFlowSpec extends munit.FunSuite {
   val subject2 = UserSubject("test2")
 
   test("unregistered login credentials should not be accepted when exchanging for token") {
-    assert(authFlow.exchangeForTokens(login).unsafeRunSync.isLeft)
+    for {
+      result <- authFlow.exchangeForTokens(login).either
+    } yield {
+      assert(result.isLeft)
+    }
```

Comment:
qui e altrove in giro non vedo le `await` o altro che rimpiazzi gli `unsafeRunSync`, chi è che scatena effettivamente l'esecuzione del valore `ZIO` ritornato?

Reply:
Questa è una buona domanda a cui non ho una risposta :sweat_smile: 
In qualche modo però la cosa avviene, dato che altrimenti i test non girerebbero proprio. Dopo averli riscritti qualche volta (scrivendo anche una versione con `await` dappertutto), ho optato per questa versione perché essendo dei test quello che mi interessano è che facciano il loro lavoro, fosse stato codice non di test avrei avuto preoccupazioni in più.